synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -73,6 +73,35 @@ export class SynopsysBridge {
     }
 
     try {
+      if (inputs.ENABLE_NETWORK_AIR_GAP) {
+        if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
+          executable = await this.setBridgeExecutablePath(
+            osName,
+            inputs.SYNOPSYS_BRIDGE_PATH
+          );
+          if (!taskLib.exist(executable)) {
+            throw new Error(
+              "synopsys_bridge_path ".concat(
+                inputs.SYNOPSYS_BRIDGE_PATH,
+                " does not exists"
+              )
+            );
+          }
+        } else {
+          executable = await this.setBridgeExecutablePath(
+            osName,
+            this.getBridgeDefaultPath()
+          );
+          if (!taskLib.exist(executable)) {
+            throw new Error(
+              "bridge_default_Path ".concat(
```

Comment:
we don't have any input variable like bridge_default_Path, can you make it Synopsys Bridge default path does not exist